shared/bash_remediation_functions/lineinfile.sh

```diff
@@ -0,0 +1,106 @@
+# Functions which minic Ansible's lineinfile module.
+# dependencies: die.sh, variables.sh
+
+function lineinfile() {
+    local path="$1"
+    local regex="$2"
+    local new_line="$3"
```

Comment:
What is this argument for? The Ansible module doesn't have it.

Reply:
@matejak I'm confused, what argument are you talking about? `new_line` is the actual line want to there.

Reply:
Yes, I meant the `new_line` argument. I thought that it is the preferred `\n` form :-) Therefore, I suggest renaming that to `line`, which is how it is called in the Ansible module.

Reply:
Oh, I see, I agree.

Reply:
Ah, so I do agree too. Must've been misremembering the docs. Will fix.